string x_state
string y_state
string z_state
float32 x
float32 y
float32 z

